src/FulfillmentLocationsList.jsx

```diff
@@ -250,7 +250,7 @@ class FulfillmentLocationsList extends React.Component {
         return (
             <div id={this._id('list-wrapper')}>
                 {this.renderFiltering()}
-                {filteredLocations.length == 0 && filteredRecentLocations.length == 0
+                {(!filteredLocations || filteredLocations.length == 0) && (!filteredRecentLocations || filteredRecentLocations.length == 0)
```

Comment:
Thanks!

Could you please also update the CHANGELOG file and bump the package version?

Reply:
I updated the changelog and version. Also I committed fix for issue when highlighting for special characters